ui-admin/src/navbar/StudySidebar.tsx

```diff
@@ -103,10 +96,6 @@ export const StudySidebar = ({ study, portalList, portalShortcode }:
         }
       </ul>}/>
       <CollapsableMenu header={'Design & Build'} content={<ul className="list-unstyled">
-        <li className="mb-2">
-          <NavLink to={studyEnvSiteContentPath(portalShortcode, study.shortcode, 'sandbox')}
-            className={sidebarNavLinkClasses} style={navStyleFunc}>Website</NavLink>
-        </li>
```

Comment:
I feel like Website should stay on the sidebar but be there regardless of the study being selected or not. I feel like I think of every important study/portal building page in Juniper being accessible from the sidebar so my brain feels weird about this page not being there. In the future, we could maybe make it clear which parts of the sidebar are study-specific and which are portal-specific. Maybe we could add titles to the sidebar to separate into sections:

```
*ourhealth Portal*
 > Website

*OurHealth Study*
> Forms & Surveys
....

Reply:
Totally agree with you, might be too premature to remove Website until the actual navigation and flow between portals<->studies is fully baked.

I'll add Website back into the original sidebar slot for the time being